src/main/java/org/triplea/client/launch/LaunchScreen.java

```diff
@@ -0,0 +1,654 @@
+    final Dimension size = frame.getSize();
+    frame.getContentPane().removeAll();
+    frame.getContentPane().add(component);
+    frame.setSize(size);
+    frame.pack();
+    frame.setSize(size);
+  }
+
+
+  private JPanel buttonPanel() {
+    return buttonPanel(null);
+  }
+
+  private JPanel buttonPanel(final Runnable backAction) {
+    return buttonPanel(backAction, null);
+  }
+
+  private JPanel buttonPanel(final Runnable backAction, final Runnable playAction) {
+    return JPanelBuilder.builder()
+        .borderEtched()
+        .flowLayout()
+        .add(JButtonBuilder.builder()
+            .biggerFont()
+            .title("Play")
+            .actionListener(playAction == null ? () -> {
+            } : playAction)
+            .enabled(playAction != null)
+            .build())
+        .add(Box.createHorizontalStrut(50))
+        .add(JButtonBuilder.builder()
+            .title("Back")
+            .actionListener(backAction == null ? () -> {
+            } : backAction)
+            .enabled(backAction != null)
+            .build())
+        .add(Box.createHorizontalStrut(50))
+        .add(JButtonBuilder.builder()
+            .title("Quit")
+            .actionListener(() -> {
+              frame.dispose();
+            })
+            .build())
+        .add(Box.createVerticalGlue())
+        .build();
+  }
+
+
+
+  private void showMoreOptions() {
+    replaceContentPaneContents(
+        JPanelBuilder.builder()
+            .addCenter(
+                JPanelBuilder.builder()
+                    .flowLayout()
+                    .add(
+                        JPanelBuilder.builder()
+                            .verticalBoxLayout()
+                            .add(Box.createVerticalStrut(40))
+                            .add(JButtonBuilder.builder()
+                                .title("Settings")
+                                .actionListener(ClientSetting::showSettingsWindow)
+                                .build())
+                            .add(Box.createVerticalStrut(40))
+                            .add(JButtonBuilder.builder()
+                                .title("Map Tools")
+                                .actionListener(() -> ProcessRunnerUtil.runClass(MapCreator.class))
+                                .build())
+                            .add(Box.createVerticalGlue())
+                            .build())
+                    .build())
+            .addSouth(buttonPanel(INSTANCE::showInitialScreen))
+            .build());
+  }
+
+  private void showMultiplayer() {
+    replaceContentPaneContents(
+        JPanelBuilder.builder()
+            .addCenter(
+                JPanelBuilder.builder()
+                    .flowLayout()
+                    .add(
+                        JPanelBuilder.builder()
+                            .verticalBoxLayout()
+                            .add(Box.createVerticalStrut(40))
+                            .add(JButtonBuilder.builder()
```

Comment:
It's a valid trigger  :smiley: 

I'll note:
- I'm avoiding simply creating intermediary variables, it can be shifting the problem and creates indirection. Done sparingly at a high level, otherwise increasing indirection is not always the best trade off in face of configuration complexity.
- working on this, I've seen better builder methods can help a lot, particularly with reducing left hand indentation.
- right hand indentation is an artifact I think of still too much logic code being in the UI configuration. With some luck, in this migration we can see the distinction between configuration, UI, and game logic code is more apparent. The goal is not a 100% split IMO, but something more like 90%/10%, so some logic can be in this configuration code, just not a lot.